plugins/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3BlobStore.java

```diff
@@ -167,9 +156,7 @@ public CannedAccessControlList getCannedACL() {
         return cannedACL;
     }
 
-    public StorageClass getStorageClass() {
-        return storageClass;
-    }
+    public StorageClass getStorageClass() { return storageClass; }
```

Comment:
I don't think you wanted to change this?

Reply:
It's a pure revert of my previous commit, where I did fix the formatting here. I'm okay with undoing the formatting fix in favor of a pure revert.

Reply:
Ok, thanks for this precision.